README.md

```diff
@@ -1573,6 +1573,39 @@ isQualtricsInterceptAvailableForUser: ({interceptId: string}) => Promise<{isAvai
 }
 ```
 
+### refreshNavBar
+
+Method that allows WebView to refresh the navigation bars that are retrieved by
+Visual Modules API
+
+```ts
+refreshNavBar: ({
+    module-id?: string,
+    product-id?: string
+}) => Promise<void>;
+```
+
+where
+
+-   `module-id` is an optional parameter
+    -   If it is not included, it means the app will refresh top and bottom bar
+    -   If it is included, it should be the same values used for Visual Modules
+        API and the app will request to refresh only the indicated bar
```

Comment:
Unrelated to the implementation of the webviewBridge method (that I see fine): currently the native FE doesn't manage module-ids, they are configured in the BE. Have you and the native devs talked about how the new approach affects their implementation (for example, they'll need to know if a module-id is used for topBar or for navBar)?

Reply:
Native app won't need to know anything about the content of the module-id, but just to pass it to the method GetNavBarData if the value is present.

Reply:
They'll need to know if the top, the bottom or both bars have to be updated, won't they?

Reply:
Maybe I'm not understanding the question. The module-id parameter in this method doesn't have a direct impact on the app. The app only needs to take this parameter (if it comes) and send it to b2p-agents when requesting the nav bar. My idea is to provide a new method in b2p-agents IDL so the app can request one concrete module-id. I assume that the current implementation implies that the app has the knowledge of what id is for the top bar a what id is for the bottom bar. And that's fine and won't change. The idea of being able to pass an id is to make less request to Kernel. 
Please, if this comment doesn't clarify, I'm happy to have a quick chat about it :)

Reply:
Let's discuss this offline, because:
>  I assume that the current implementation implies that the app has the knowledge of what id is for the top bar a what id is for the bottom bar

The answer to this is "no". Only b2p-agents BE has that knowledge as of today

Reply:
Checked offline. native code won't need to know the correlation, the BE can return the structure that is being returned as of today, containning only the element that changes (nav bar or tab bar)